app/src/main/java/br/com/tick/teira/ui/theme/Theme.kt

```diff
@@ -60,9 +61,13 @@ fun TeiraTheme(
         }
     }
 
-    MaterialTheme(
-        colorScheme = colorScheme,
-        typography = Typography,
-        content = content
-    )
+    CompositionLocalProvider(
+        LocalSpacing provides Spacing()
```

Comment:
A ideia é criar aqui várias dependências para espaços, cores, fontes e etc?

Reply:
Eu inicialmente ainda comecei a criar para os `Sizing` para usar em dimensões de cards e afins.
Mas depois desisti por causa do naming. Fico perdido. 🤷‍♂️ 
Mas sim, diria que a ideia é fazer algo idêntico para cores e fonts, e chamar-mos os nossos "padrões" em vez dos do Material (se for essa a intenção).